pkg/virt-handler/vm.go

```diff
@@ -820,6 +822,9 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 			}
 			vmi.Status.Conditions = append(vmi.Status.Conditions, liveMigrationCondition)
 		}
+		if hasHotplug {
+			liveMigrationCondition.Status = k8sv1.ConditionFalse
+		}
 		if liveMigrationCondition.Status == k8sv1.ConditionTrue {
 			vmi.Status.Conditions = append(vmi.Status.Conditions, liveMigrationCondition)
 		}
```

Comment:
It doesn't appear that the migration condition get's set to false when hasHotPlug is set. 

Maybe this logic needs to be restructured as

```
if hasHotplug {
			liveMigrationCondition = v1.VirtualMachineInstanceCondition{
				Type:    v1.VirtualMachineInstanceIsMigratable,
				Status:  k8sv1.ConditionFalse,
				Message: "LiveMigration of VMI's with hotplugged disks is not supported"
				Reason:  v1.VirtualMachineInstanceReasonDisksNotMigratable,
			}
			vmi.Status.Conditions = append(vmi.Status.Conditions, liveMigrationCondition)
}
```

Reply:
I fixed the logic so it can change as needed.